Tiltfile

```diff
@@ -1,79 +1,95 @@
 #!/usr/bin/env python
 load('ext://restart_process', 'docker_build_with_restart')
 load('ext://color', 'color')
+load('ext://dotenv', 'dotenv')
 
 analytics_settings(True) # Enable telemetry dialogue in web UI
 disable_snapshots()      # Disable TiltCloud Snapshots
 version_settings(True)   # Enable 'new version' banner
 
-# parse Tiltfile config
-config.define_string(name="env", args=False, usage="This argument defines the build env (dev or release)")
-settings = config.parse()
-environment=settings.get('env', "dev") #dev or release
+# Load environment variables from .env file
+dotenv()
 
-# load ingresses
-k8s_yaml('./cluster/charts/fuel-local/crds/traefik-resources.yaml')
-k8s_kind('IngressRoute')
+# Build publisher image with proper configuration for Minikube
+custom_build(
```

Comment:
With this we are able to use our local `fuel-streams-publisher` Dockerfile instead of getting the one from our Github, this would be better to test local modification in the publisher